File: MyFlow.Data/Models/Flowchart.cs
namespace MyFlow.Data.Models
{

    public interface IFlowchart 
    {
        public int Id { get; set; }
        public string? FlowName { get; set; }
        public string? FlowEname { get; set; }
        public int? FlowType { get; set; }
        public string? AdminUser { get; set; }
        public string? Target { get; set; }
        public int? Close { get; set; }
        public string? TagFormat { get; set; }
        public string? TitleFormat { get; set; }
    }

    public partial class Flowchart : IFlowchart, IDataModel
    {
        public int Id { get; set; }
        public string? FlowName { get; set; }
        public string? FlowEname { get; set; }
        public int? FlowType { get; set; }
        public string? AdminUser { get; set; }
        public string? Target { get; set; }
        public int? Close { get; set; }
        public string? TagFormat { get; set; }
        public string? TitleFormat { get; set; }

    }
}

File: MyFlow.View/mock/flowchart.js
const testCases = [
  {
    "id": 0,
    "name": "差勤",
    "children": [
      {
        "id": 1,
        "name": "請假單",
        "children": null
      }
    ]
  },
  {
    "id": 1,
    "name": "總務",
    "children": [

    ]
  }
];
  
const mock = [
  {
      url: '/api/Flowchart',
      method: 'get',
      response: ({ query ,body }) => {
        return testCases;
      },
  },
  {
    url: '/api/Flowchart/Admin',
    method: 'get',
    response: ({ query ,body }) => {
        return testCases;
    },
  },
  {
      url: '/api/Flowchart/:id',
      method: 'get',
      response: ({ query, body, headers }) => {
        return {
          Id: 1,
          FlowName: '請假單',
          FlowType: 1,
          AdminUser: 'TestUser',
          TagFormat: 'A000',
          TitleFormat: 'A000'
        };
    },
  }
]
  
  export default mock